pgdrive/scene_manager/spawn_manager.py

```diff
@@ -120,48 +142,69 @@ def get_target_vehicle_configs(self, seed=None):
         if len(target_agents) > 1:
             for real_idx, idx in enumerate(target_agents):
                 v_config = self.target_vehicle_configs[idx]["config"]
+                v_config = self._randomize_position_in_slot(v_config)
```

Comment:
Oh so you create the concept like "init_config" and "run_time_instantized_config'? Great idea!